web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -1610,6 +1526,23 @@ class AppVersionHistory extends Component<Props, State> {
               ?.downloadingVersion
           }
           adminConsoleMetadata={this.props.adminConsoleMetadata}
+          onWhyNoGeneratedDiffClicked={(rowVersion: Version) =>
+            this.toggleNoChangesModal(rowVersion)
```

Comment:
some of this stuff is getting prop drilled. I will probably add a modal provider but I need to get a handle on what they are and how they work first 

moved from: 
https://github.com/replicatedhq/kots/blob/d8ab2b5cc500cd296ab19fb82dc5f7ef05404196/web/src/components/apps/AppVersionHistory.tsx#L561-L571